app/controllers/welcome_controller.rb

```diff
@@ -0,0 +1,7 @@
+class WelcomeController < ApplicationController
+  def index
+    if !current_user.nil?
+      redirect_to dashboard_path
+    end
+  end
+end
```

Comment:
@LHJE @dcoleman21 Is there anything on the welcome page that we do want logged in users to have access to?

Reply:
Not really?  It's just sort of a "Here is the basic information about what this app is", a slide show of Dani's PDF stuff (which in and of itself is not a resource I would say, more of an enticer), a link to log in, a link to the Learn More page, which the user would also have access to via the nav bar.  But we can defo chat about that before merging.